frontend/javascripts/oxalis/view/action-bar/merge_modal_view.tsx

```diff
@@ -141,40 +101,51 @@ class _MergeModalView extends PureComponent<Props, MergeModalViewState> {
     });
   };
 
-  handleMergeProject = async (event: React.SyntheticEvent, isLocalMerge: boolean) => {
+  handleMergeProject = async (event: React.SyntheticEvent) => {
     event.preventDefault();
     const { selectedProject } = this.state;
 
     if (selectedProject != null) {
-      if (isLocalMerge) {
-        const annotation = await getAnnotationCompoundInformation(
-          selectedProject,
-          APIAnnotationTypeEnum.CompoundProject,
-        );
-        this.mergeAnnotationIntoActiveTracing(annotation);
-      } else {
-        const url =
-          `/api/annotations/CompoundProject/${selectedProject}/merge/` +
-          `${this.props.annotationType}/${this.props.annotationId}`;
-        this.merge(url);
-      }
+      const annotation = await getAnnotationCompoundInformation(
+        selectedProject,
+        APIAnnotationTypeEnum.CompoundProject,
+      );
+      this.mergeAnnotationIntoActiveTracing(annotation);
+    }
+  };
+
+  handleImportProject = async (event: React.SyntheticEvent) => {
+    event.preventDefault();
+    const { selectedProject } = this.state;
+
+    if (selectedProject != null) {
+      const annotation = await getAnnotationCompoundInformation(
+        selectedProject,
+        APIAnnotationTypeEnum.CompoundProject,
+      );
+      this.mergeAnnotationIntoActiveTracing(annotation);
+    }
+  };
+
+  handleMergeExplorativeAnnotation = async (event: React.SyntheticEvent) => {
+    event.preventDefault();
+    const { selectedExplorativeAnnotation } = this.state;
+
+    if (selectedExplorativeAnnotation != null) {
+      const url =
+        `/api/annotations/Explorational/${selectedExplorativeAnnotation}/merge/` +
+        `${this.props.annotationType}/${this.props.annotationId}`;
+      this.merge(url);
```

Comment:
how about renaming the merge-method to `createMergedAnnotation` or something like that to make its behavior clearer?